### DISCUSSION TITLE
Provide a figma

### DISCUSSION THREAD

**Discussion by eznix86** [founded-labs/react-native-reusables#14] [Ideas]
I know it is not the scope of this repository, but most people in the mobile development space have a design system where they design beforehand what they expect as a design.

Turning the current components into figma components would be beneficial for the project.

**Comment by mrzachnugent** [comment]
For the web components, here's an awesome Figma community file:
https://www.figma.com/community/file/1333848637032170061

**Comment by rhinck** [comment]
This would be a really nice feature to have. Our team prefers using react-native-reusables, but we’re currently building a new design system and it makes the decision between rnr and glue-ui harder—mainly because glue-ui ships with a built-in Figma file.

We don’t have dedicated designers, so we end up doing both design and coding ourselves. We’d like to stick with rnr, but without an official Figma library we’d have to recreate the components from scratch on the design side, which adds overhead and increases the risk of drift between design and code.

Having an official rnr Figma file (even just a minimal v1 with tokens + a handful of primitives) would make adoption much easier for teams like ours.

**Comment by seangray-dev** [comment]
Yes a Figma file for all of mobile components would be super beneficial!

**Comment by Byevenes** [comment]
What else is needed to promote having a figma for this? I'm in the same situation, and honestly, I'd like to continue using RNR, but I need help for my designers. On the front end, the client uses the Flowbite design system, but on mobile we're behind.

**Comment by Amirthananth** [comment]
I built one figma UI kit for my team to work on our product using reactnativereusables. I'll share it here if someone needs to use it.

let me know if you need me to add some more examples.

https://www.figma.com/community/file/1640811989795288896

**Reply by Amirthananth** [reply]
@mrzachnugent I've created a PR to add this to the docs for easy reference - https://github.com/founded-labs/react-native-reusables/pull/540.

Please take a look when you get time
